src/pfe/portal/modules/FileWatcher.js

```diff
@@ -585,13 +586,14 @@ module.exports = class FileWatcher {
     }
   }
 
-  async writeWorkspaceSettings(workspaceSettings) {
+  async writeWorkspaceSettings(address, namespace) {
     let retval;
     try{
-      retval = await filewatcher.writeWorkspaceSettings(workspaceSettings);
+      retval = await filewatcher.writeWorkspaceSettings(address, namespace);
       this.logFWReturnedMsg(retval);
     } catch (err) {
-      log.error(err);
+      log.error(`Error in writeWorkspaceSettings`);
```

Comment:
Is this error meant to remain here? If so, should we include the contents of the err object?

Reply:
Error is logged in `imagePushRegistry.route.js`

```
 retval = await user.writeWorkspaceSettings(address, namespace);
    }
    res.status(retval.statusCode).send(retval);
  } catch (error) {
    log.error(error);
    const workspaceSettings = {
      statusCode: 500,
      imagePushRegistryTest: false,
      msg: error.message
    }
    res.status(500).send(workspaceSettings);
  }
```

Didnt want multiple places to have same err being logged